src/platform/graphics/webgpu/webgpu-graphics-device.js

```diff
@@ -649,6 +649,22 @@ class WebgpuGraphicsDevice extends GraphicsDevice {
         });
     }
 
+    setupTimeStampWrites(passDescr, name) {
```

Comment:
Isn't the typical abbreviation of description desc rather than descr? 😄

Reply:
yep done, thanks